log/Logger.js

```diff
@@ -1,123 +1,24 @@
-const merge = require('lodash.merge');
-const moment = require('moment');
-const { logging, outputTypes } = require('./config');
-const levels = logging.log4js.levels;
-
-const RequestTracing = require('./tracing/requestTracing');
-
-let userConfig = {};
-
-function isBlank (str) {
-  return !str || str.trim().length === 0;
+const moment = require('moment')
+const winston = require('winston')
+
+const container = new winston.Container()
+const tsFormat = () => (moment().format('YYYY-MM-DDTHH:mm:ssZ'))
+
+function transports (label) {
+  return [new winston.transports.Console({
+    level: process.env.LOG_LEVEL || 'INFO',
```

Comment:
Log level has to be lower case. Default 'INFO' causes that logs are not being produced at all.

I will fix that